basic-algorithms.js

```diff
@@ -1,7 +1,60 @@
 // Names and Input
+var hacker1 = 'chloe';
+console.log('The driver\'s name is ' + hacker1);
 
+var hacker2 = prompt('What\'s your name?');
+console.log('The navigator\'s name is ' + hacker2);
 
 //Conditionals
 
+if (hacker1.length > hacker2.length) {
+  console.log('The Driver has the longest name, it has ' + hacker1.length + ' characters');
+} else if(hacker1.length < hacker2.length) {
+  console.log('Yo, navigator got the longest name, it has ' + hacker2.length + ' characters');
+} else{
+  console.log('Wwow, you both got equally long names, ' + hacker1.length + ' characters!!');
+}
+
+console.log(hacker1.split('').join(' ').toUpperCase());
+
+var reverse = '';
+for (var i = hacker1.length - 1 ; i>=0; i--) {
+  reverse += hacker1[i];
+}
+console.log(reverse)
+
+  if (hacker1 < hacker2) {
```

Comment:
Fix indentation. All the code that goes below should go one more space to the left.